src/os/net/request.js

```diff
@@ -220,6 +221,20 @@ os.net.Request.METHOD_DELETE = 'DELETE';
 os.net.Request.METHOD_HEAD = 'HEAD';
 
 
+/**
```

Comment:
I have a task where I handle an error differently based on the status code.  I thought having the codes in a central location will facilitate doing similar things in the future.